### DISCUSSION TITLE
wiki.js

### DISCUSSION THREAD

**Discussion by bytearchive** [mkdocs/mkdocs#3159] [Q&A]
can mkdocs be integrated to wiki.js?
or can it be modified to have wiki + users ?

**Comment by pawamoy** [comment]
You'll need to provide much more details to your question. Without more information, the (short) answer is no.
You can serve your MkDocs pages behind a proxy that deals with users and authorization, but you won't be able to render dynamic contents based on the current user (unless you want to do that in Javascript entirely, client-side, or to modify pages in the proxy before sending them back to the client).
